extensions/interactions/base.py

```diff
@@ -143,6 +143,25 @@ def _stats_log_template(self):
             return '{{answer}}'
 
     @property
+    def rules(self):
+        if self._cached_rules is not None:
+            return self._cached_rules
+
+        rules_index_dict = json.loads(
```

Comment:
Can we load the ``rules.json`` file somewhere else 1 time? It might be more performance benefiting to cache the JSON file in 1 location, then just lookup the subdict for this property instead of loading the JSON file for each interaction type. WDYT?